runtime/layers/function/Dockerfile

```
@@ -8,6 +8,6 @@ COPY php.ini /opt/bref/etc/php/conf.d/bref.ini
 # Remove PHP-FPM
 RUN rm /opt/bref/sbin/php-fpm /opt/bin/php-fpm
 
-# Build the final image from the lambci image that is close to the production environment
-FROM lambci/lambda:build-provided.al2
+# Build the final image from the amazon image that is close to the production environment
+FROM public.ecr.aws/lambda/provided:al2
 COPY --from=0 /opt /opt
```

@Nyholm to keep BC regarding filesystem write proection we could add the 2 following instructions into this Dockerfile, bu also in `runtime/layers/fpm/Dockerfile`
```suggestion
COPY --from=0 /opt /opt
RUN echo "bref:x:1000:1000:Bref:/:/sbin/nologin" >> /etc/passwd \
  && chown bref:1000 /tmp && \
  chmod 700 /tmp
USER bref
```

---

No, I think it is a good thing we dont have the write protection

---

Ok perfect, looks also good to me to keep this PR as is.

---

so @mnapoli I guess we can move forward and merge :angel:

---

Yep I'm intending to merge this, just took some holidays this week :P I'm avoiding non-urgent work this week, will do that next week.

Thanks for all the clarification! I also agree (for now) about not replicating the filesystem restrictions: most apps will write to the project directory (if they ever write anything) which is mounted, so users have control over making read-only that volume.

---

Thx - nothing urgent - enjoy holliday time!

---

I'll make another PR to upgrade to php 8.0.1 as it fixes an important tokenizer issue that would help using php-cs-fixer with php8 see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5415#issue-549599811.